### PR TITLE
Add Solos Space

### PR DESCRIPTION
**Pull request by crazyrabbitLTC**
#### Hi! What is your PR about?
Adding images to our space. 

- [x] Add images for your space
- [ ] Add custom domain for your space
- [ ] Add alias for migrate your space
- [ ] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: spaces/solos.eth/domains.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "snapshot.solos.so": "solos.eth"
+}
```
Comment (gokaiorg) [inline spaces/solos.eth/domains.json:3]: - You have to add your subdomain in the list of the file https://github.com/snapshot-labs/snapshot-spaces/blob/master/spaces/domains.json
- You should not add domains.json files in your space, please delete it.

**Review by gokaiorg — CHANGES_REQUESTED** [review]
The images file size should not exceed 50KB. Please reduce their weight.

**Comment by gokaiorg** [comment]
I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/solos.eth

**Comment by crazyrabbitLTC** [comment]
Sorry just did it!

Thank you!

Dennison Bertram

www.dennisonbertram.com

NYC: +1 718 644 4272

**Pull request merged**